lib/wibox/container/only_on_screen.lua

```diff
@@ -0,0 +1,145 @@
+---------------------------------------------------------------------------
+--
+-- @author Uli Schlachter
+-- @copyright 2017 Uli Schlachter
+-- @classmod wibox.container.only_on_screen
```

Comment:
Should this be in `awful.widget`? This is on the line between Awesome specific and generic. If we abide to the rule where "wibox is independent from Awesome", then it makes little sense there. On the other hand, the context have the screen, so the coupling is technically already present.

I am fine both way, but I think it's worth thinking about it.